packages/google-pay-integration/src/gateways/google-pay-gateway.ts

```diff
@@ -78,11 +78,13 @@ export default class GooglePayGateway {
             company = '',
             phone = '',
             email,
+            customFields = [],
         } = this._paymentIntegrationService.getState().getBillingAddress() || {};
 
         return {
             ...this._mapToAddressRequestBody(billingAddress, company, phone),
             email: email || response.email,
+            customFields,
```

Comment:
I would like to suggest to pass this parameter to `_mapToAddressRequestBody` function where we return object based on `AddressRequestBody` interface. And cover it by a test to make sure we don't face a similar problem in the future